src/main/java/org/terracotta/build/plugins/docker/DockerBuildPlugin.java

```diff
@@ -205,6 +206,11 @@ private void configureBuildDefaults(Project project, DockerBuildExtension docker
     dockerBuild.getMetadata().put("gradle.build.host", getLocalHostName(project));
     dockerBuild.getMetadata().put("gradle.build.dir", project.getRootDir().getAbsolutePath());
     dockerBuild.getMetadata().put("gradle.build.project", project.getPath());
+
+    BuildInfo buildInfo = BuildInfo.getOrInstall(project);
```

Comment:
It feels like this should be being retrieved through the project extension to decouple the way the BuildDataPlugin is implemented from how things are consumed.

Reply:
I will ping you today for more explanation because you are diving too deep in Gradle for me there :P

Reply:
outdated